Minor fixes

emmanuel-bv opened the pull request:
Fixing an apostrophe issue, cleaning up the create options and clarifying OS / container types

azurecla commented:
Hi __@pablissima__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (paulettm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

emmanuel-bv committed:
Minor fixes

Fixing an apostrophe issue, cleaning up the create options and clarifying OS / container types
